roles/openshift_logging/defaults/main.yml

```diff
@@ -12,6 +12,8 @@ openshift_logging_install_logging: False
 openshift_logging_purge_logging: False
 openshift_logging_image_pull_secret: ""
 
+openshift_logging_es5_techpreview: False
```

Comment:
we are only going to allow tech preview for origin, we need to guard against a non-origin and this being used

Reply:
We will have our image tags be "techpreview" or something similar